app.js

```diff
@@ -75,6 +77,8 @@ app.use(st({ path: './public', url: '/public' }));
 marked.setOptions({ sanitize: true });
 app.locals.marked = marked;
 
+console.log("hello world 2");
```

Comment:
####  <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

http.createServer uses HTTP which is an insecure protocol and should not be used in code due to cleartext transmission of information. Data in cleartext in a communication channel can be sniffed by unauthorized actors. Consider using the https module instead.

**Path**: https://github.com/trailblazing-dx/tiago_pr_playground/blob/433c1a339cbf64b5f324c6f159381da09371a200/public/css/screen.css#L23

[CWE-352](https://cwe.mitre.org/data/definitions/352.html) | [Priority score 560](https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score)

<details>

<summary>Data flow: 3 steps</summary>

https://github.com/trailblazing-dx/tiago_pr_playground/blob/855a1fa7877e00fd6c9f4c15ef769d8f5ea3b0cf/app.js#L52

https://github.com/trailblazing-dx/tiago_pr_playground/blob/855a1fa7877e00fd6c9f4c15ef769d8f5ea3b0cf/app.js#L75

https://github.com/trailblazing-dx/tiago_pr_playground/blob/855a1fa7877e00fd6c9f4c15ef769d8f5ea3b0cf/app.js#L80 

</details>